### ISSUE TITLE
When I put "flingContainer.setAdapter(arrayAdapter);" into the OnCreate(), it can be seen when the activity start! But when I put "flingContainer.setAdapter(arrayAdapter);" into the OnClickListener of a button , the view cannot be seen!

### ISSUE THREAD

**Issue by jzqiao** [Diolor/Swipecards#189]

**Comment by QAFlexsin** [comment]
I have same issues can you help me I want show my view after response but my view not show. My adapter constructor call but adapter getView and getCount method not call.

**Comment by jzqiao** [comment]
Have you called the method notifyDataSetChanged() of the adapter?

**Comment by QAFlexsin** [comment]
yes I have called notifyDataSetChanged() after get response from retrofit library but it's not working.

**Comment by jzqiao** [comment]
Can you pack the code and send it to jzqiao@outlook.com？ If solved the problem, I would feedback to you as soon as possible.

**Comment by JonatanPanduro** [comment]
I got the same problem using setAdapter in other function on my Fragment, getCount() and getView() were never called on adapter, but using notifyDataSetChanged() in this way
/*arrayAdapter = new CustomBaseAdapter(getActivity(), list);
flingContainer.setAdapter(arrayAdapter);
arrayAdapter.notifyDataSetChanged();
*/
it works
